src/hooks/useNodeId.ts

```diff
@@ -0,0 +1,8 @@
+import { useParams } from 'react-router-dom';
+
+const useNodeId = () => {
```

Comment:
Travis failed on the last commit, I think you just need to add the return type here